components/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/APIConsumerImpl.java

```diff
@@ -120,6 +120,16 @@ public BusinessOwner  getBusinessOwner(int businessOwnerId) throws AppManagement
         return appMDAO.getBusinessOwner(businessOwnerId, tenantId);
     }
 
+    /**
+     * Returns business owner Ids by a prefix of business owner name.
```

Comment:
Formatting and Java doc comments issues are fixed in https://github.com/wso2/carbon-appmgt/pull/539/files